tools/f18/f18.cpp

```diff
@@ -123,14 +125,14 @@ bool ParentProcess() {
 void Exec(std::vector<char *> &argv, bool verbose = false) {
   if (verbose) {
     for (size_t j{0}; j < argv.size(); ++j) {
-      std::cerr << (j > 0 ? " " : "") << argv[j];
+      llvm::errs() << (j > 0 ? " " : "") << argv[j];
     }
-    std::cerr << '\n';
+    llvm::errs() << '\n';
   }
   argv.push_back(nullptr);
   execvp(argv[0], &argv[0]);
-  std::cerr << "execvp(" << argv[0] << ") failed: " << std::strerror(errno)
-            << '\n';
+  llvm::errs() << "execvp(" << argv[0] << ") failed: " << llvm::sys::StrError(errno)
+               << '\n';
```

Comment:
It looks like this needs reformatting. Maybe other files too.

Reply:
This is clang-format-7! I can re-run to see if it will change.

Reply:
Done!